CONTRIBUTE.md

```diff
@@ -23,15 +23,55 @@ or navigate to the respective folder and `yarn add <package>`
 
 ### From terminal
 
-1. `git fetch upstream`
-2. `git checkout main` if not already on main branch
-   `git merge upstream/main`
-3. `git push origin main` to update the forked repo
+**1.** If anyone contribute to this repository, then the changes will not reflect in your local repository. For that:
+
+**2.** Setup a reference(remote) to the original repository to get all the changes from the remote.
+
+```
+   git remote add upstream https://github.com/webdev-20/TLL-hacktoberfest-2022.git
+```
+
+**3.** Check the remotes for this repository.
+
+```
+   git remote -v
+```
+
+**4.** Fetching from the remote repository will bring in its branches and their respective commits.
+
+```
+   git fetch upstream
+```
+
+**5.** Make sure that you're on your main branch.
```

Comment:
this also works if you want to pull latest changes to your current working branch which is not main, resolve conflicts if you have to